### ISSUE TITLE
extrautils2 Fallen Angel Ring

### ISSUE THREAD

**Issue by xenoflot** [rwtema/extrautilities_old_issues#931]
Hi,

I built (not spawned) an Angel Ring today. I'm finding the ring is dropping me quite frequently. Sometimes it drops me after only a few seconds, otherwise it seems to be around the 40 second mark. I have plenty of power (56gp, 12 from solar, the rest from lava) and only an inactive resonator. Not taking any falling damage, fortunately. Most of my testing has been in the nether but it was dropping me in the overworld, too. Nothing in the logs that seems relevant.

Side issue: The ring does not fit in a baubles slot.

Are these intended behaviour? If not, here's your bug report :)

extrautils2-1.10-alpha-1.0.0
forge 12.180.0.2000
MC 1.10.2

**Comment by xenoflot** [comment]
Further info: After returning to base, I found there were still 8 blocks of stoneburnt in the resonator. I removed the stoneburnt and broke the resonator. I got the achievement when I picked up the resonator. From that point, the ring worked just fine.

Thinking that it may be that some kind of periodic check (and thus power fluctuation) while the resonator's inventory isn't empty, I put some more polished stone in and then set the time to night. Unfortunately(!) my ring has been stable for 4 minutes now, so it's not that.

The only clue thus far is that the resonator was placed prior to the crafting of the ring and hasn't been buggy since I broke and re-placed it. Sorry but this could be hard to reproduce!
